src/renderer/components/cluster-manager/clusters-menu.tsx

```diff
@@ -75,6 +84,12 @@ export class ClustersMenu extends React.Component<Props> {
             label: _i18n._(t`Remove`),
           },
           ok: () => {
+            const wsId = workspaceStore.currentWorkspace.id;
```

Comment:
We can use observable directly `workspaceStore.currentWorkspaceId` in every place where `workspaceStore.currentWorkspace.id` located.

Reply:
Good catch! Will update and commit